public/app/features/alerting/unified/components/notification-policies/useNotificationPolicyRoute.ts

```diff
@@ -1,27 +1,43 @@
 import memoize from 'micro-memoize';
 
-import { routingTreeApi } from 'app/features/alerting/unified/api/notificationPoliciesApi';
+import { generatedRoutesApi as routingTreeApi } from 'app/features/alerting/unified/openapi/routesApi.gen';
```

Comment:
Two imports here - this one and below with the types. They can be combined together

Reply:
Good catch! I wonder why the linter(s) didn't catch this one